Cargo.toml

```diff
@@ -1,23 +1,37 @@
 [workspace]
-members = ["ceramic-kubo-rpc", "ceramic-one"]
+members = [
+    "core",
+    "event",
+    "ceramic-kubo-rpc",
+    "ceramic-one"
+]
 
 [workspace.dependencies]
 anyhow = "1"
 async-stream = "0.3"
 async-trait = "0.1"
 ceramic-one = { path = "./ceramic-one" }
 dag-jose = "0.1"
+cid = { version = "0.10.1", features = ["serde-codec"] }
+env_logger = "0.10"
 futures-util = "0.3"
 iroh-api = { git = "https://github.com/nathanielc/beetle", branch = "expose-api-client" }
 iroh-embed = { git = "https://github.com/nathanielc/beetle", branch = "expose-api-client" }
 iroh-metrics = { git = "https://github.com/nathanielc/beetle", branch = "expose-api-client" }
 iroh-rpc-client = { git = "https://github.com/nathanielc/beetle", branch = "expose-api-client" }
 libipld = "0.15"                                                                                 # use same version as Iroh
 libp2p = { version = "0.50", default-features = false }                                          # use same version as Iroh
+log = "0.4"
```

Comment:
Similar question for `log` re `env_logger`. I'd like to exclusively use the `tracing` analogs for this code base. Thoughts?